gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint-group/create/api-endpoint-group-create.component.spec.ts

```diff
@@ -288,8 +288,8 @@ describe('ApiEndpointGroupCreateComponent', () => {
     describe('Endpoint group name validation', () => {
       const EXISTING_ENDPOINT_GROUP = fakeEndpointGroupV4({
         type: 'kafka',
-        name: 'Existing endpoint group',
-        endpoints: [fakeKafkaMessageEndpoint({ name: 'Existing endpoint' })],
+        name: 'Existing endpoint group ',
```

Comment:
Feels strange that the existing value is not trimmed. I would prefer reversing the test and adding a trailing space when setting the value with the harness

Reply:
I guess it's a legacy test then 😅 . I'll add some more tests to cover different scenarios 👍